.gitignore

```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+
+yarn.lock
```

Comment:
@nirajunify you've mentioned we've to keep package-lock.json under version control, Does same is applied for yarn.lock or not?

Reply:
Yes. Yarn.lock should be under version control. @krajasekhar-unify please update the code

Reply:
Updated